runtime/bindings/python/src/pyopenvino/core/infer_request.cpp

```diff
@@ -65,9 +65,9 @@ void regclass_InferRequest(py::module m) {
             self._start_time = Time::now();
             self._request.infer();
             self._end_time = Time::now();
-            Containers::InferResults results;
-            for (auto& out : self._outputs) {
-                results.push_back(self._request.get_tensor(out));
+            Containers::InferVec results;
+            for (const auto& out : self._outputs) {
+                results.push_back(std::make_pair(out, self._request.get_tensor(out)));
             }
             return results;
```

Comment:
I think there is a way to create such OrderedDict on pybind side.

Reply:
Python 3.6+ dict respect order, so I don't know if there is a need for it.

Reply:
So we can return `py::dict` from pybind instead of copying it over-again?

Reply:
Updated.